radio/src/gui/480x272/model_setup.cpp

```diff
@@ -1076,6 +1090,11 @@ bool menuModelSetup(event_t event)
           if (MULTIMODULE_HAS_SUBTYPE(moduleIdx))
             lcdDrawMultiSubProtocolString(MODEL_SETUP_4TH_COLUMN, y, moduleIdx, g_model.moduleData[moduleIdx].subType, menuHorizontalPosition==2 ? attr : 0);
         }
+#endif
+#if defined(AFHDS3)
+        else if (isModuleAFHDS3(moduleIdx)) {
+          lcdDrawTextAtIndex(MODEL_SETUP_4TH_COLUMN, y, STR_AFHDS3_PROTOCOLS, g_model.moduleData[moduleIdx].rfProtocol, (menuHorizontalPosition==1 ? attr : 0));
```

Comment:
Maybe lcdNextPos + 20 is a nicer choice than MODEL_SETUP_4TH_COLUMN which is quite far right ?